plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2064,271 @@ def coupling_parameter(
+
+            :math:`x^{α\backslashβ}=\frac{m_{β}v_{α}^{2}}{2k_B T_{β}}`,
+
+            :math:`ψ\left(x\right)=\frac{2}{\sqrt{\pi}}\int_{0}^{x}t^{\frac{1}{2}}e^{-t}dt`,
+
+            :math:`ψ'\left(x\right)=\frac{dψ}{dx}`,
+
+        and :math:`\lambda_{\alpha \beta}` is the Coulomb logarithm for the collisions,
+        :math:`n_\beta` is the number density of the field particles, :math:`v_\alpha` is
+        the speed of the test particles relative to the field particles, :math:`k_B` is Boltzmann's
+        constant, and :math:`T_\beta` is the temperature of the field particles.
+
+        For values of x<<1 (the 'slow' or 'thermal' limit) or x>>1 (the 'fast' or 'beam' limit),
+        :math:`\psi` asymptotes to zero or one respectively. For simplified expressions in these limits
+        we encourage the curious reader to refer to p. 31 of :cite:t:`nrlformulary:2019`
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies_using_temperature = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies_using_temperature.momentum_loss
+        <Quantity 1.83701432e+11 Hz>
+
+        >>> v_a = 1e7 * u.cm / u.s
+        >>> frequencies_using_velocity = CollisionFrequencies(
+        ...     "e-", "e-", v_a=v_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies_using_velocity.energy_loss
+        <Quantity -9.69828719e+15 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+        """
+
+        # Note: This function uses CGS units internally to coincide with our references.
+        # Input is taken in MKS units and then converted as necessary. Output is in MKS units.
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle).to(u.cm / u.s)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+    @cached_property
+    def _mass_ratio(self):
+        return self.test_particle.mass / self.field_particle.mass
+
+    @cached_property
+    def momentum_loss(self):
+        """
+        The momentum loss for the specified particles.
```

Comment:
"momentum loss rate due to collisions" 

This is just the rate at which momentum is lost, not the amount of momentum lost, which would be the solution to this differential equation (which would be exponential energy loss, e.g.   v = e^(- \nu_ab t) )

Same for the others: e.g. rate of transverse velocity diffusion due to collisions , rate of energy loss.